Source/WebKit/NetworkProcess/PrivateClickMeasurement/PrivateClickMeasurementManager.h

```diff
@@ -30,6 +30,7 @@
 #include "PrivateClickMeasurementClient.h"
 #include "PrivateClickMeasurementManagerInterface.h"
 #include "PrivateClickMeasurementStore.h"
+#include <WebCore/PrivateClickMeasurement.h>
```

Comment:
Do we need this or can what's needed be forward declared?

Reply:
Ah, good catch — this include was actually redundant altogether since it's already included by another one of the above headers.